.utils/deploy.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+# This file is used to deploy yttest to an S3 bucket. It
+# expects the S3 bucket name in an environment variable
+# $YTTEST_BUCKET
+
+set -ex
+
+if [ ! -d "test-materials" ] || [ ! -d "2019" ]; then
+    echo "Can't find test-materials/ or 2019/ directory. Are you running"\
```

Comment:
Out of curiosity, how much work will it be in the future to change from `2019` to eg. `2020`? Shall we have a constant at the top for year in use?

Reply:
> Out of curiosity, how much work will it be in the future to change from 2019 to eg. 2020? 

The work involved would be file a PR to change it from "2019" to "2020".

> Shall we have a constant at the top for year in use?

I don't have an opinion on this tbh. Please feel free to update this PR with a constant if you'd like to do so.